# Array variable message of VehicleStatus
Header header
VehicleStatus[] VehiclesDetected